appengine/cloudsql-mysql/Gemfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google, Inc
```

Comment:
I'm not sure how you feel about the 2017 here, because this is an old sample being transitioned for Cloud SQL PostgreSQL.